src/io_uring.c

```diff
@@ -0,0 +1,92 @@
+#include "io_uring.h"
+
+#ifdef HAVE_LIBURING
+#include <liburing.h>
+
+#define IO_URING_DEPTH 256 /* io_uring instance queue depth. */
+
+static size_t io_uring_queue_len = 0;
+
+void initIOUring(void) {
+    if (server.io_uring_enabled) {
```

Comment:
Let's have the caller check `server.io_uring_enabled` so the entire io_uring module doesn't need to take a dependency *back* on server.h. The dependency should go in one direction.

```suggestion
```

Reply:
Agree. We only need `#include "zmalloc.h"`.